io.openems.edge.battery.fenecon.home/src/io/openems/edge/battery/fenecon/home/FeneconHomeBattery.java

```diff
@@ -405,214 +394,231 @@ public default void setBmsControl(BmsControl value) throws OpenemsNamedException
 				.unit(Unit.VOLT) //
 				.accessMode(AccessMode.READ_WRITE) //
 				.text("CV Point")),
-		BCU_BMS_SOFTWARE_VERSION(Doc.of(OpenemsType.INTEGER) //
+		BCU_2_BMS_SOFTWARE_VERSION(Doc.of(OpenemsType.INTEGER) //
 				.accessMode(AccessMode.READ_WRITE) //
-				.text("Bcu Bms Software Version")),
+				.text("Bcu 2 Bms Software Version")),
+		BCU_3_BMS_SOFTWARE_VERSION(Doc.of(OpenemsType.INTEGER) //
+				.accessMode(AccessMode.READ_WRITE) //
+				.text("Bcu 3 Bms Software Version")),
 		BCU_BMS_HARDWARE_VERSION(Doc.of(OpenemsType.INTEGER) //
 				.accessMode(AccessMode.READ_WRITE) //
 				.text("Bcu Bms Hardware Version")),
-		BCU_STATUS_ALARM(Doc.of(OpenemsType.BOOLEAN) //
+		
+		//BCU Status Flags
+		STATUS_ALARM(Doc.of(OpenemsType.BOOLEAN) //
 				.accessMode(AccessMode.READ_WRITE) //
 				.text("BCU Status Alarm")),
-		BCU_STATUS_WARNING(Doc.of(OpenemsType.BOOLEAN) //
+		STATUS_WARNING(Doc.of(OpenemsType.BOOLEAN) //
```

Comment:
Some of these 'Boolean' Channels sound like they should actually be StateChannels with Info/Warning/Fault.